deepchem/models/tests/test_optimizers.py

```diff
@@ -1,3 +1,5 @@
+from haiku import Linear
```

Comment:
Did you intend to add this?  I don't think you use it anywhere.  Since it requires JAX, it should only be imported inside functions that are marked for JAX only.

Reply:
yea must have been added by the IDE by default and just slipped in. Will change it